doc/01_general/04_architecture.md

```diff
@@ -0,0 +1,319 @@
+# Architecture of vehicle agent
+
+**Summary:** This page gives an overview over the general architecture of the vehicle agent.
+The document contains an overview over all [nodes](#overview) and [topics](#topics).
+
+---
+
+## Authors
+
+Marco Riedenauer, Simon Erlbacher, Julian Graf
+
+## Date
+
+25.11.2022
+
+
+---
+<!-- TOC -->
+
+* [Architecture of vehicle agent](#architecture-of-vehicle-agent)
+    * [Author](#authors)
+    * [Date](#date)
+    * [Overview](#overview)
+    * [Perception](#perception)
+        * [Obstacle Detection and Classification](#obstacle-detection-and-classification)
+        * [Lane detection](#lane-detection)
+        * [Traffic Light Detection](#traffic-light-detection)
+        * [Traffic Signs Detection](#traffic-signs-detection)
+        * [Prediction](#prediction)
+        * [Localization](#localization)
+    * [Planning](#planning)
+        * [Preplanning](#preplanning)
+        * [Decision Making](#decision-making)
+        * [Local path planning](#local-path-planning)
+    * [Acting](#acting)
+        * [Path following](#path-following)
+        * [Velocity control](#velocity-control)
+        * [Emergency](#emergency)
+    * [Visualization](#visualization)
+    * [Topics](#topics)
+
+<!-- TOC -->
+
+## Overview
+
+The vehicle agent is split into three major components: [Perception](#Perception), [Planning](#Planning)
+and [Acting](#Acting).
+A separate node is responsible for the [visualization](#Visualization).
+The topics published by the Carla bridge can be
+found [here](https://carla.readthedocs.io/projects/ros-bridge/en/latest/ros_sensors/).
+The msgs necessary to control the vehicle via the Carla bridge can be
+found [here](https://carla.readthedocs.io/en/0.9.8/ros_msgs/#CarlaEgoVehicleControlmsg)
+
+![Architecture overview](../00_assets/Übersicht.jpg)
+
+## Perception
+
+The perception is responsible for the efficient conversion of raw sensor and map data into a useful
+environment representation that can be used by the [Planning](#Planning) for further processing.
+
+Further information can be found [here](../03_research/02_perception/Readme.md).
```

Comment:
Maybe "Further information regarding the perception can be found..." would be more precise.

Reply:
Fixed in 3f8ef00137e56173f50a933439c7fa21ba3738a6